doc/content/reference/telemetry/identity_server.md

```diff
@@ -0,0 +1,89 @@
+---
+title: "Identity Server Telemetry"
+description: "Description of everything that the telemetry procedure collects regarding the IdentityServer"
+aliases: "/telemetry/identity-server"
+---
+
+[The Things Stack]({{<ref "/the-things-stack">}})'s Identity Server collects telemetry data, which is used to help us understand how users interact with the product. For example, the current data helps to have a better insight on the user's usage of organizations. Telemetry collection is enabled by default and does not collect personally identifiable information. Telemetry collection is enabled by default and does not collect personally identifiable information. The following section section explains what information is collected and how it is anonymised.
```

Comment:
No need to put ref to TTS, but there's need to put ref to Identity Server :) also, use {{% tts %}} always instead of "The Things Stack"

Reply:
I also see this whole part is the same as for the CLI telemetry doc. Why don't you put it to /telemetry then? It's also weird that you stressed how this data collection proces is **completely anonymous** for the CLI, but not here. What do you refer to when you say "current data"?

Reply:
Is a very small difference but for some users they might and can change what values the Stack will use to generate the userID, meaning that they are in control of what is used to generate the hash.

OBS: In this comment and in the previous comment I used `userID` in multiple phrases but it is meant to reference the Identifier used when collecting information, it is not linkable to the User by itself. Meaning that is not possible to do `Hash(ID) -> User Information` but it is possible to do `UserInformation -> TelemetryID` in the Stack.

Reply:
Removed the **completely anonymous** words from the CLI description, if you think this paragraph is good enough we could remove it from the CLI and IS descriptions and leave it on the `index.md`

Reply:
Great, I understand the difference now so this makes sense. And yeah, it would be good to move it to index.md